Classes/Controller/BackendModuleController.php

```diff
@@ -155,13 +187,50 @@ public function sendJobPostingUrisToGoogleApiAction($siteNodeName, $siteDimensio
             array_push($nodesUriToUpdate, $this->uriBuilderService->getLastUri());
         }
 
+        if (empty($nodeToUpdate)) {
+            $message = $this->translateById('error.nodesToUpdateIsEmpty');
+            $this->addFlashMessage('', $message, Message::SEVERITY_ERROR);
+            $this->info($message);
+        }
+
         foreach ($nodesUriToUpdate as $nodeUriToUpdate) {
-            $googleResponse = $this->jobPublishingService->updateJobPosting($nodeUriToUpdate);
+            try {
+                $this->info('Try to update ' . $nodeUriToUpdate);
+                $this->jobPublishingService->updateJobPosting($nodeUriToUpdate);
+                $message = $this->translateById('successful.nodesUpdated');
+                $this->addFlashMessage('', $message, Message::SEVERITY_OK);
+                $this->info('JobPosting with URL: ' . $nodeUriToUpdate . ' updated...');
+            } catch (Google_Service_Exception $error) {
+                $message = $this->translateById('error.googleServiceException');
+                $this->addFlashMessage('', $message . " \r\n " . $error->getMessage() , Message::SEVERITY_ERROR);
+                $this->info('Error: ' . $message . " \r\n - Status code: " . $error->getCode() . ' - Message: ' . $error->getMessage());
+            }
         }
 
-        $this->view->assignMultiple(array(
-            'googleResponse' => $googleResponse,
-        ));
+        if ($this->showGoogleClientConfiguration) {
```

Comment:
The attribute here should be rather "logGoogleClientConfiguration" than "showGoogleClientConfiguration" as it only logs.

Reply:
*Updated "showGoogleClientConfiguration" to "logGoogleClientConfiguration"*.  
There is no __toString Implementation.